charts/prometheus-adapter/templates/custom-metrics-apiserver-psp.yaml

```diff
@@ -0,0 +1,71 @@
+{{- if .Values.psp.create -}}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "k8s-prometheus-adapter.fullname" . }}
+  labels:
+    app: {{ template "k8s-prometheus-adapter.name" . }}
+    chart: {{ template "k8s-prometheus-adapter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  {{- if .Values.hostNetwork.enabled }}
+  hostNetwork: true
+  {{- end }}
+  fsGroup:
+    rule: RunAsAny
+  runAsGroup:
+    rule: MustRunAs
+    ranges:
+    - max: 0
```

Comment:
Would prefer RunAsAny since we probably want to switch this to non root group soon.

Reply:
Ok later we can change to RunAsNonRoot